### ISSUE TITLE
Social Media Link Preview

### ISSUE THREAD

**Issue by datenreisen** [DigitaleGesellschaft/jekyll-theme-conference#13]
Link previews would be nice.

A minimalist variant would be (code in html/head):

```
<meta property="og:title" content="XXX" />
<meta property="og:description" content="XXX" />
<meta property="og:image" content="https://www.digitale-gesellschaft.ch/uploads/2022/01/Winterkongress_Logo_sm.gif" />
<meta name="twitter:card" content="summary_large_image" />

```
XXX replaced by \<title\> respectively \<meta name=description content\>
Image: locally stored (in /assets/...)

**Comment by lorenzschmid** [comment]
- Twitter: [Summary Card](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/summary) based on [Cards Markup](https://developer.twitter.com/en/docs/twitter-for-websites/cards/overview/markup)
- Facebook/WhatsApp/iMessage/Telegram: [Open Graph Markup](https://developers.facebook.com/docs/sharing/webmasters/)

**Comment by lorenzschmid** [comment]
Fixed in 7754dd791832f2b679427df4c68f49a14100025a
